### DISCUSSION TITLE
Insufficient privileges

### DISCUSSION THREAD

**Discussion by afn24805** [Set-OutlookSignatures/Set-OutlookSignatures#101] [Q&A] [answered]
I executed Create-EntraApp.ps1 as per the guide to create an internal app. When running it, I get an error saying I don't have sufficient privileges even though I logged in as the Global Admin of or tenant. What else can I look at to fix the error?

**Comment by GruberMarkus** [comment]
Please post the script you are using and the text output of it.

Alternatively, you can create the Entra ID app manually by following the instructions in 'default graph config.ps1'.

**Comment by GruberMarkus** [comment, marked as answer]
This could be a scope consent problem. Change the following line in the sample script and try again.

From:
```
Connect-MgGraph -NoWelcome
```

To:
```
Connect-MgGraph -Scopes 'Application.ReadWrite.All' -NoWelcome
```

**Reply by afn24805** [reply]
That did the trick. Thanks Markus!

**Comment by afn24805** [comment]
It looks like it needs Azure Active Directory P1 or P2 for this? We only have Azure Active Directory Basic for Education (we're a public school).
